speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backendUnitTests/RecorderTests.cpp

```diff
@@ -255,11 +267,11 @@ TEST(RecorderTests, WriteDecisions)
     for (int i = 0; i < 10; i++)
     {
         DecisionTuple decisionTuple;
-        decisionTuple.SetAccel(random.NextFloat(0, 1));
-        decisionTuple.SetBrake(random.NextFloat(0, 1));
-        decisionTuple.SetGear(random.NextInt(0, 10));
-        decisionTuple.SetSteer(random.NextFloat(0, 1));
-        expectedDecisionsData << bits(timestamp) << bits(decisionTuple.GetSteer()) << bits(decisionTuple.GetAccel()) << bits(decisionTuple.GetBrake()) << bits(static_cast<float>(decisionTuple.GetGear()));
+        decisionTuple.SetAccelDecision(random.NextFloat(0, 1));
+        decisionTuple.SetBrakeDecision(random.NextFloat(0, 1));
+        decisionTuple.SetGearDecision(random.NextInt(0, 10));
+        decisionTuple.SetSteerDecision(random.NextFloat(0, 1));
+        expectedDecisionsData << bits(timestamp) << bits(decisionTuple.GetSteerAmount()) << bits(decisionTuple.GetAccelAmount()) << bits(decisionTuple.GetBrakeAmount()) << bits(static_cast<float>(decisionTuple.GetGearAmount()));
```

Comment:
This line is way too long, spread it over multiple lines, for example every `<<`

Reply:
Whenever I do that the code coverage tool fails on Linux (but not on Windows). I would like some further instructions on how I could solve that.

Reply:
Something like this should work, if that doesn't you can just add the expectedDecisionsData stream in front of every `<<`.
```suggestion
        expectedDecisionsData << bits(timestamp)
                              << bits(decisionTuple.GetSteerAmount())
                              << bits(decisionTuple.GetAccelAmount())
                              << bits(decisionTuple.GetBrakeAmount())
                              << bits(static_cast<float>(decisionTuple.GetGearAmount()));
```